.eslintrc

```diff
@@ -23,6 +23,7 @@
         "./private/**/*.js"
       ],
       "env": {
+        "browser": false,
```

Comment:
Wat is de reden dat het op false wordt gezet?

Reply:
Dit is een stukje met overrides voor node files. Die nam die value over van de algemene config maar dit moet gewoon op false in node files, anders denkt hij dat dingen als `fetch`, `window`, `document` enzo bestaan.